schema/verify/types/emission_form_data.sql

```diff
@@ -0,0 +1,13 @@
+-- Verify ggircs-portal:types/emission_form_data on pg
+
+begin;
+
+do $$
+  begin
+    assert (
+      select true from pg_catalog.pg_type where typname = 'emission_form_data'
```

Comment:
I had a doubt, but the column is indeed `typname`, and not `typename`. That's an interesting choice from postgres...